locals.yml.default

```diff
@@ -23,6 +23,13 @@ dashboard_enable_pegasus: true
 #use_my_apps: true
 
 
+# Allows you to change your locale locally.
```

Comment:
I'm not super familiar with the way this flag works, so feel free to correct the language I'm using in the explanation comment!

Reply:
I'd make it slightly more specific; something like "Allows your local server to render non-English locales, defaults to false. If false, choosing a locale other than English will have no effect."